Makefile

```diff
@@ -2,8 +2,8 @@
 
 Brick.ttl: bricksrc/*.py bricksrc/*.ttl bricksrc/definitions.csv generate_brick.py
 	mkdir -p extensions
-	python generate_brick.py
-	cd shacl && python generate_shacl.py
+	python3.10 generate_brick.py
+	cd shacl && python3.10 generate_shacl.py
 
```

Comment:
Can you change these back to `python` from `python3.10`? We need them generic so we can test across multiple versions of Python

Reply:
Done